lib/bundler/cli.rb

```diff
@@ -437,6 +441,12 @@ def env
       Env.new.write($stdout)
     end
 
+    if ENV["BUNDLE_PLUGIN"]
```

Comment:
I did that in  4a1baea, but some specs were failing because when called here (during the loading of `Cli` class)  `Bundler.settings` gets initialized before `CLI` class gets initialized.

So some options like `--gemfile` don't get properly set. Initializing `Settings` calls `Bundler.root` which caches the root path determined without respecting `--gemfile` option (which is set in [cli#initialize](https://github.com/bundler/bundler/blob/master/lib/bundler/cli.rb#L21)).

So reverted it cf284c0. I tried digging but failed to fix the problem.